app/go_to_date/templates/index.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <link href="{{ url_for('static', path='/styles.css') }}" rel="stylesheet">
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<body>
+    <button class="open-button" onclick="openForm()">Go to a specific date</button>
```

Comment:
Use JavaScript's `addEventListener`, don't use inline `onclick` event

Reply:
Remove the `openForm` (and don't mark as resolved before you actually resolve it)